multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1342,10 +1343,11 @@ def _fit(
         with apply_log_filter(log_filter):
```

Comment:
Is it possible for us to filter the warning here?

Reply:
They will remove this flag in the v1.10.0, which I guess will come soon.